backend_modules/libvirt/host/user_data.yaml

```
@@ -152,6 +170,12 @@ zypper:
       baseurl: http://${ use_mirror_images ? mirror : "download.suse.de/ibs"}/SUSE/Products/SLE-SERVER/12-SP4/x86_64/product/
       enabled: 1
       autorefresh: 1
+    - id: tools_pool_repo
+      baseurl: http://${ use_mirror_images ? mirror : "download.opensuse.org"}/repositories/systemsmanagement:/Uyuni:/Stable:/SLE12-Uyuni-Client-Tools/SLE_12/
```

I think it would be more accurate to use SUSE Manager Tools repos with SLES images (also with the others below).
```suggestion
      baseurl: http://${ use_mirror_images ? mirror : "download.suse.de/ibs"}/Devel:/Galaxy:/Manager:/4.3:/SLE12-SUSE-Manager-Tools/SLE_12/
```